code/datums/tutorial/marine/role_specific/hospital_corpsman_basic.dm

```diff
@@ -0,0 +1,1090 @@
+	manual_message_to_player("The Dummy has taken some kind of brute damage. Stand next to them, and click on their body with your <b>Health Analyzer</b> in hand to scan them.")
+	add_highlight(marine_dummy, COLOR_GREEN)
+	update_objective("Click on the Dummy with your Health Analyzer")
+	RegisterSignal(marine_dummy, COMSIG_LIVING_HEALTH_ANALYZED, PROC_REF(scanner_3))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/scanner_3(datum/source, mob/living/carbon/human/attacked_mob)
+	SIGNAL_HANDLER
+
+	if(attacked_mob == tutorial_mob)
+		return
+
+
+	UnregisterSignal(marine_dummy, COMSIG_LIVING_HEALTH_ANALYZED)
+
+	manual_message_to_player("Good. By looking at the Health Analyzer interface, we can see they have 5 brute damage on their chest.")
+	manual_message_to_player("A chemical called <b>Bicaridine</b> is used to heal brute damage over time.")
+	manual_message_to_player("<b>Bicaridine</b> is primarily given in pill form.")
+	register_tutorial_step(CALLBACK(src, PROC_REF(brute_tutorial_3_pre)))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/brute_tutorial_3_pre(obj/item/storage/pill_bottle)
+	SIGNAL_HANDLER
+
+	handle_pill_bottle(marine_dummy, "Bicaridine", "Bi", "11", /obj/item/reagent_container/pill/bicaridine)
+	RegisterSignal(tutorial_mob, COMSIG_MOB_TUTORIAL_HELPER_FAIL, PROC_REF(brute_tutorial_3_pre), TRUE)
+	RegisterSignal(tutorial_mob, COMSIG_MOB_TUTORIAL_HELPER_RETURN, PROC_REF(brute_pill_fed))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/brute_pill_fed()
+	SIGNAL_HANDLER
+
+	UnregisterSignal(tutorial_mob, COMSIG_MOB_TUTORIAL_HELPER_FAIL)
+	UnregisterSignal(tutorial_mob, COMSIG_MOB_TUTORIAL_HELPER_RETURN)
+
+	manual_message_to_player("When administered in pill form, chemicals take a few seconds to be digested before they can enter the patients bloodstream, and heal damage.")
+	manual_message_to_player("Medications administered when a patient is dead will not heal damage until the patient has been revived.")
+
+	update_objective("")
+
+	register_tutorial_step(CALLBACK(src, PROC_REF(brute_tutorial_4)))
+
+/datum/tutorial/marine/role_specific/hospital_corpsman_basic/proc/brute_tutorial_4()
+	SIGNAL_HANDLER
+
+
+	marine_dummy.rejuvenate()
+	marine_dummy.apply_damage(5, BRUTE, "chest")
+
+	var/obj/item/reagent_container/hypospray/autoinjector/bicaridine/skillless/one_use/brute_injector = new(loc_from_corner(0, 4))
+	add_to_tracking_atoms(brute_injector)
+	add_highlight(brute_injector)
+
+	manual_message_to_player("Medicines can also be directly injected into the bloodstream using <b>Autoinjectors</b>, allowing the medication to begin work immediately, without the need to be digested first.")
+	manual_message_to_player("Click on the <b>Bicaridine Autoinjector</b> with an empty hand to pick it up, and then click on the Dummy while standing next to it to administer an injection")
+	update_objective("Pick up, and inject the Dummy with the Bicaridine Autoinjector")
```

Comment:
objectives should end in a period, no need for the comma, autoinjector isn't a proper noun.